frontend/src/components/configurate-episode/components/create-episode-form/create-episode-form.tsx

```diff
@@ -25,19 +25,27 @@ type Props = {
   onSubmit: (payload: EpisodeFormPayload) => void;
   payload?: EpisodeFormPayload;
   imageUrl?: string;
+  fileUrl?: string;
 };
 
 const selectTypeOptions: Option[] = getOptions(Object.values(EpisodeType));
-const selectStatusOptions: Option[] = getOptions(Object.values(EpisodeStatus));
+const selectStatusOptions: Option[] = getOptions(
+  Object.values(EpisodeStatus).filter((status) => status != EpisodeStatus.LIVE),
```

Comment:
Let's use strict comparison

```suggestion
  Object.values(EpisodeStatus).filter((status) => status !== EpisodeStatus.LIVE),
```